src/test/java/bio/terra/janitor/service/pubsub/TrackedResourceSubscriberTest.java

```diff
@@ -100,4 +103,56 @@ public void nack() {}
                 PubsubMessage.newBuilder().setData(ByteString.copyFromUtf8("bad json")).build(),
                 consumer));
   }
+
+  // See https://broadworkbench.atlassian.net/browse/CORE-104
```

Comment:
I did confirm this test failed in the expected way before `FAIL_ON_UNKNOWN_PROPERTIES` -- and it passes now.